envision/web/src/helpers/state_unpacker.js

```diff
@@ -33,9 +33,9 @@ const WorldState = Object.freeze({
   SCENARIO_ID: 1,
   SCENARIO_NAME: 2,
   TRAFFIC: 3,
-  BUBBLES: 4,
-  SCORES: 5,
-  EGO_AGENT_IDS: 6,
+  BUBBLES: 5,
+  SCORES: 6,
+  EGO_AGENT_IDS: 7,
```

Comment:
Why was this needed?

Reply:
This fixed the bubble rendering issue. There is an empty array in front of the bubbles in 4th position. This causes the bubbles, scores and ego_agent_ids arrays capture the wrong information. (The bubbles capture the empty array, scores are actually bubbles information, and ego_agent_ids are actually scores.)

Reply:
I am not sure what the empty array is used for. For now I did not see any usage of that array, but I am not sure whether it contains information in other scenarios, so I just move one position forwards for the bubbles, scores and ego_agent_ids.

Reply:
I just looked into this, and it's because traffic signals were added at index 4 without updating this enum (see `_format_state()` in `envision/data_formatter.py`). I think we should add a line like `SIGNALS: 4,` here so this is clear in the future.

Reply:
I will add `SIGNALS: 4` in this branch, thanks!